app.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
Why delete this file?

Reply:
I deleted this by mistake. I shall add it again.